### DISCUSSION TITLE
关于PP-OCRv4 中英文字识别模型使用的训练数据问题

### DISCUSSION THREAD

**Discussion by JackLee1** [PaddlePaddle/PaddleOCR#13261] [Q&A]
准备对PP-OCRv4 中英文字识别进行调优，请问一下PP-OCRv4 自带的中英文字识别模使用是哪些公开的训练数据集？

**Comment by SWHL** [comment]
v4的好像没写英文版，中文版凑合看吧 → [docs](https://paddlepaddle.github.io/PaddleOCR/ppocr/blog/PP-OCRv4_introduction.html)

v3的 → [PP-OCRv3: More Attempts for the Improvement of Ultra Lightweight OCR System](https://arxiv.org/abs/2206.03001)
